### DISCUSSION TITLE
Issue Closed.

### DISCUSSION THREAD

**Discussion by ShubhamDalmia** [jtablesaw/tablesaw#1157] [Q&A] [answered]
Issue Closed.

**Comment by lwhite1** [comment]
I don’t see anything to suggest this is a tablesaw issue.   Have you looked
at
https://bugs.openjdk.org/browse/JDK-8192647

You might check to ensure the runner config matches the jar environment,
and maybe try a newer/different Java version.

Other than that all I can say is maybe the table is too big for the memory
available.

Sorry that’s about all I can say with the information provided.

**Comment by lwhite1** [comment, marked as answer]
Unfortunately, I cannot debug a garbage collection deadlock based on a few
print outs. Even if i could Tablesaw does not control GC threads.

FWIW, up above you say you're using Tablesaw 43.0, but the pom says 41.0.
Maybe try 43,
